### DISCUSSION TITLE
Minimal view tiles heading text not visible

### DISCUSSION THREAD

**Discussion by mail02ankit** [Lissy93/dashy#695] [Help and Support] [answered]
Hi All,

I started using Dashy recently. It has been really good. Thanks for this nice application.

I found a problem with my installation. Somehow, when I switched to minimal view the text in the heading of tiles is not visible. For the default view it works fine. I have tried switching themes and color schemes, it did not help.

Any help will be appreciated,
Thanks.

![minimal_dashy](https://user-images.githubusercontent.com/19859282/171756946-995e80bb-251a-47c0-997a-b7808e698299.png)

**Comment by lissy93** [comment, marked as answer]
Hey @mail02ankit 
This looks like a bug, so I've opened a ticket for it: #697

Sorry about that. I _think_ it's caused by when short on room only icons are displayed, but in this case, you don't have any section items, so nothing is displayed. But I'll look into it, and hopefully get a fix pushed out in the next update :)

**Reply by mail02ankit** [reply]
No worries, thanks.
